src/custom-elements/src/components/dropdown/dropdown.tsx

```diff
@@ -148,6 +163,14 @@ export class Dropdown {
     );
   }
 
+  displayBlock() {
+    this._dropdownMenuElement.style.display = 'block';
+  }
+
+  displayNone() {
+    this._dropdownMenuElement.style.display = 'none';
+  }
+
```

Comment:
These could be merged into one single method and a parameter. Thoughts?

Reply:
agree, I suggest `setDisplay()` which would accept `'block' | 'none'`